addons/account/views/account_move_views.xml

```diff
@@ -362,8 +362,10 @@
                     <filter string="Cash" name="cash" domain="[('move_id.journal_id.type', '=', 'cash')]" context="{'default_journal_type': 'cash'}"/>
                     <filter string="Miscellaneous" domain="[('move_id.journal_id.type', '=', 'general')]" name="misc_filter" context="{'default_journal_type': 'general'}"/>
                     <separator/>
-                    <filter string="Payable" domain="[('account_id.internal_type', '=', 'payable')]" help="From Payable accounts" name="payable"/>
-                    <filter string="Receivable" domain="[('account_id.internal_type', '=', 'receivable')]" help="From Receivable accounts" name="receivable"/>
+                    <filter string="Payable" domain="[('account_id.internal_type', '=', 'payable'), ('account_id.non_trade', '=', False)]" help="From Trade Payable accounts" name="trade_payable"/>
+                    <filter string="Receivable" domain="[('account_id.internal_type', '=', 'receivable'), ('account_id.non_trade', '=', False)]" help="From Trade Receivable accounts" name="trade_receivable"/>
+                    <filter string="Non Trade Payable" domain="[('account_id.internal_type', '=', 'payable'), ('account_id.non_trade', '=', True)]" help="From Non Trade Receivable accounts" name="non_trade_payable"/>
+                    <filter string="Non Trade Receivable" domain="[('account_id.internal_type', '=', 'receivable'), ('account_id.non_trade', '=', True)]" help="From Non Trade Receivable accounts" name="non_trade_receivable"/>
```

Comment:
these lines are too long (> 120 characters). Can you indent it plz? :)

Reply:
Is it mandatory? I personally think this is easier to read/see the difference between the lines...

Reply:
Imo, the following is the best indent because you can see directly all attributes.
```xml
<filter name="non_trade_receivable"
        string="Non Trade Receivable"
        domain="[('account_id.internal_type', '=', 'receivable'), ('account_id.non_trade', '=', True)]"
        help="From Non Trade Receivable accounts"/>
```
 Also, it makes new diff easier to read. For example, if "Payable" was correctly indented at the first place:
```xml
    <filter string="Payable"
-           domain="[('account_id.internal_type', '=', 'payable')]"
+           domain="[('account_id.internal_type', '=', 'payable'), ('account_id.non_trade', '=', False)]"
            help="From Payable accounts"
-           name="payable"/>
+           name="trade_payable"/>
```
Blaming from git is also easier.
It's my personal opinion so ignore it if you don't agree